arch/mx.rst

```diff
@@ -1,13 +1,10 @@
 .. _mx:
 
-Masterless Mode (beta)
```

Comment:
Let's rename this. We've moved away from 'master' to 'coordinator'. What are our options?
- MX Architecture? (Stick with MX branding)
- Citus MX

Reply:
I'll stick with "Citus MX"